### DISCUSSION TITLE
Zoom meeting links

### DISCUSSION THREAD

**Discussion by DharaniSureshBabu** [cu-esiil-edu/2024-data-short-course#43] [Q&A]
@eculler @nquarder  Where can I find the all 4 weeks zoom meeting links and office hours links ? Sorry, if I missed it.

**Comment by nquarder** [comment]
Hi @DharaniSureshBabu - here are the links to the zoom meetings for office hours:

[Weds 2-4pm MST in the US](https://cuboulder.zoom.us/j/99130800989) <- click here to join
[Thurs 12-2pm MST in the US](https://cuboulder.zoom.us/j/97773318270) <- click here to join

The links for the recorded sessions are now in the course syllabus. There is a section under the **Curriculum** heading for each week for recordings that you should see. All three for last week are up as well as the one from yesterday. 

See here for course syllabus page - https://cu-esiil-edu.github.io/2024-data-short-course/
